content/en/docs/tasks/access-application-cluster/access-cluster.md

```diff
@@ -96,6 +98,26 @@ $ curl $APISERVER/api --header "Authorization: Bearer $TOKEN" --insecure
 }
 ```
 
+Using `jsonpath` approach:
+
+```shell
+$ APISERVER=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')
+$ TOKEN=$(kubectl get secret $(kubectl get serviceaccount default -o jsonpath='{.secrets[0].name}') -o jsonpath='{.data.token}' | base64 --decode )
```

Comment:
nit: is the command prompt needed?
nit: possibly add a line between setting the variables and calling the curl command, followed by another line.
nit: I'm okay to combine the output of multiple kubectl cmds into a single variable, but it may be easier to parse if
separated like in the original PR:
$ TOKEN=$(kubectl get secret $SECRET_NAME -o jsonpath='{.data.token}' | base64 --decode )

nit: can the extra space after decode be removed?

Reply:
cool, I address your PR comments.